stack/corepersistence/queue/src/main/java/org/apache/usergrid/persistence/queue/settigs/QueueIndexingStrategy.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
.../settigs/QueueIndexingStrategy -- this is a duplicate file

Reply:
removed